src/index.ts

```diff
@@ -16,6 +16,7 @@ import {
 } from "./componentprocessor";
 import { Parser } from "./parser";
 import { Interpreter, ExecutionOptions, defaultExecutionOptions } from "./interpreter";
+import { _resetTestData } from "./extensions";
```

Comment:
Instead of importing and re-exporting the function, we should be able to do this:
```suggestion
export { resetTestData } from "./extensions";
```